sky/packages/sky/lib/src/widgets/scrollable.dart

```diff
@@ -94,15 +94,13 @@ abstract class Scrollable extends StatefulComponent {
 
   Widget build() {
     return new GestureDetector(
+      onVerticalDragStart: _handleDragStart,
       onVerticalDragUpdate: _getDragUpdateHandler(ScrollDirection.vertical),
       onVerticalDragEnd: _getDragEndHandler(ScrollDirection.vertical),
+      onHorizontalDragStart: _handleDragStart,
```

Comment:
Adding these callbacks unconditionally will cause the GestureDetector to try to detect them all the time, which means every Scrollable will be looking for both a horizontal and a vertical drag start.